src/content/guides/payment-terminals.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Payment Terminal Integration
+description: How to integrate a payment terminal with Centrapay APIs.
+nav:
+  path: Reference/Centrapay Integrations
+  order: 1
+---
+
+Integrating a merchant terminal with Centrapay APIs requires creating, cancelling, voiding and refunding [Payment Requests](https://docs.centrapay.com/api/payment-requests) on behalf of Merchants using a “merchant terminal” API key.
+
+## API Keys
+
+To create API keys, you first need to get in touch with Centrapay to be issued an Integrator Account and an “account owner” API key. An “account owner” is a special kind of role that allows you to manage your account.
+
+You can use this key to [Create an API Key](https://docs.centrapay.com/api/api-keys#create-an-api-key) with the “merchant terminal” role. A “merchant terminal” key has a role that can create, cancel, void and refund Payment Requests on behalf of merchants.
+
+> You are responsible for the safety of your API keys
+
+## Merchant Configs
+Centrapay [Merchant Configs](https://docs.centrapay.com/api/merchant-configs) represent an available set of configured payment methods that can be utilized by one or more payment terminals by a Merchant.
+
+Merchants and merchant configs are managed by Centrapay. The only details you need to be able to create payment requests on their behalf are the `merchantId` and `merchantConfigId` (aka `clientId`). Centrapay will send you these details through whatever channel works for you. Please get in touch to let us know.
+
+In the future Centrapay will allow an integrator to on-board their own merchants through APIs using integrator API keys.
+
+## Example Flows
+
+See [Payment Flows](/guides/payment-flows) for an overview of the API calls required for different payment flows.
```

Comment:
Just checking that this does not need to be a full URL?

Reply:
Nope, works like this if it's the same site.
Can't do it if it's linking from new docs to old docs as it breaks locally as they're not the same base path.